open-api/rest-catalog-open-api.py

```diff
@@ -629,7 +638,7 @@ class TransformTerm(BaseModel):
     term: Reference
 
 
-class ReportMetricsRequest2(CommitReport):
+class ReportMetricsRequest1(CommitReport):
```

Comment:
why do we need these changes?

Reply:
It seems to happen when I run the `python3 rest-catalog-open-api.py` I can undo this